proxy/ControlMatcher.h

```diff
@@ -136,7 +137,16 @@ class HttpRequestData : public RequestData
   inkcoreapi sockaddr const *get_ip();
   inkcoreapi sockaddr const *get_client_ip();
 
-  HttpRequestData() : hdr(NULL), hostname_str(NULL), api_info(NULL), xact_start(0), incoming_port(0), tag(NULL), internal_txn(false)
+  HttpRequestData()
+    : hdr(NULL),
+      hostname_str(NULL),
+      api_info(NULL),
+      xact_start(0),
+      incoming_port(0),
+      tag(NULL),
+      internal_txn(false),
+      cache_info_lookup_url(NULL),
```

Comment:
I added the cache_info_lookup_url since it was mentioned to be needed for TS-4025 by @jrushford in one of the comments above. If we are not going to use it (but use only the parent selection URL instead), then I can remove it. But I would like to hear from @zwoop or @jrushford to confirm.